test/routes/startOrOpenSaved.route.test.js

```diff
@@ -6,24 +6,44 @@ const Code = require('code')
 const server = require('../../server')
 
 const DOMParser = require('xmldom').DOMParser
+const Application = require('../../src/models/application.model')
+const CookieService = require('../../src/services/cookie.service')
 
-let validateTokenStub
+let generateCookieStub
+let validateCookieStub
+let applicationSaveStub
 
 let routePath = '/start/start-or-open-saved'
+let applicationId = 'TEST_APPLICATION_ID'
+
+const fakeCookie = {
+  applicationId: 'my_application_id',
+  authToken: 'my_auth_token'
+}
 
 lab.beforeEach((done) => {
   // Stub methods
-  validateTokenStub = server.methods.validateToken
-  server.methods.validateToken = () => {
-    return 'my_token'
+  generateCookieStub = CookieService.generateCookie
+  CookieService.generateCookie = (reply) => {
+    return fakeCookie
+  }
+  validateCookieStub = CookieService.validateCookie
+  CookieService.validateCookie = (cookie) => {
+    return true
   }
 
+  applicationSaveStub = Application.prototype.save
+  Application.prototype.save = (authToken) => {
+    return applicationId
```

Comment:
As discussed, to ensure we match the actual method, this needs to return `undefined` (though might be a discussion point as to whether `model.save()` should be returning anything before we add more 🤔 )

Reply:
I have modified the stub for now since we don't currently need the save() method to return anything. If anything goes wrong with the save, the try...catch should pick it up. We might need to revisit this later once we have a firmer understanding of how we need to handle Dynamics errors.